unlock-app/.jest/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  rules: {
+    'import/no-extraneous-dependencies': [
```

Comment:
Wait, wasn't that part of another PR? Did you rebase?

Reply:
HA! 1 PR 1 purpose ;) do you mind moving that to another one? I know this is annoying but it makes things so much easier down the line.

Reply:
yep, marked as WIP so no one is tempted to approve it